functests/performance/updating_profile.go

```diff
@@ -133,6 +133,28 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernel is default when realTimeKernel field does not exist", func() {
+			conditionUpdating := machineconfigv1.MachineConfigPoolUpdating
+
+			if profile.Spec.RealTimeKernel == nil || *profile.Spec.RealTimeKernel.Enabled == true {
+				Skip("Skipping test - This test expects RT Kernel to be disabled. Found it to be enabled or nil.")
+			}
+
+			By("Applying changes in performance profile")
+			profile.Spec.RealTimeKernel = nil
+			Expect(testclient.Client.Update(context.TODO(), profile)).ToNot(HaveOccurred())
+
+			Expect(profile.Spec.RealTimeKernel).To(BeNil())
+			By("Checking that the updating MCP status will consistently stay false")
+			Consistently(func() corev1.ConditionStatus {
+				return getMcpStatus(testutils.RoleWorker, conditionUpdating)
+			}, 5, 5).Should(Equal(corev1.ConditionFalse))
```

Comment:
the 1st time arg is the duration, the second the interval. So I think the 1st should be something like 30?

Reply:
I thought 25 seconds should be good enough. Do we want to change it? Maybe 15, 4 ?

Reply:
Done.